Makefile

```diff
@@ -265,28 +265,34 @@ install-tools:
 generate:
 	go generate ./...
 
-.PHONY: artifacts-darwin
-artifacts-darwin:
-	mkdir -p _artifacts
-	GOOS=darwin GOARCH=amd64 make clean binaries
-	$(TAR) -C _output/ -czvf _artifacts/lima-$(VERSION_TRIMMED)-Darwin-x86_64.tar.gz ./
-	GOOS=darwin GOARCH=arm64 make clean binaries
-	$(TAR) -C _output -czvf _artifacts/lima-$(VERSION_TRIMMED)-Darwin-arm64.tar.gz ./
+.PHONY: artifacts-darwin artifact-darwin-aarch64 artifact-darwin-arm64 artifact-darwin-x86_64
+artifacts-darwin: artifact-darwin-x86_64 artifact-darwin-arm64
+artifact-darwin-arm64: ENVS=GOOS=darwin GOARCH=arm64
```

Comment:
I was not aware that you can have target-specific environment variables in Makefiles, and kind of find it confusing how they are even applied to building prerequisites (and if multiple targets share a prerequisite, then it is kind of random which env variables are in effect).

But it looks like this isn't a problem in our case, so I guess it is fine. Still makes my head spin a little though.

Reply:
Even within the `artifacts-darwin` target before this change, there were inconsistencies in the way parameters were passed to `$(TAR)`, so I thought it would be best to unify them as much as possible.
```Makefile
.PHONY: artifacts-darwin
artifacts-darwin:
	mkdir -p _artifacts
	GOOS=darwin GOARCH=amd64 make clean binaries
	$(TAR) -C _output/ -czvf _artifacts/lima-$(VERSION_TRIMMED)-Darwin-x86_64.tar.gz ./
	GOOS=darwin GOARCH=arm64 make clean binaries
	$(TAR) -C _output -czvf _artifacts/lima-$(VERSION_TRIMMED)-Darwin-arm64.tar.gz ./
```